.gitignore

```diff
@@ -9,6 +9,10 @@ Full_DDL.sql
 # C extensions
 *.so
 
+# MERGING
```

Comment:
Should we still be ignoring these files?

Reply:
Was having issues with re base just changed this